concepts/05 Widgets/SelectBox/00 Getting Started with SelectBox/00 Getting Started with SelectBox.md

```diff
@@ -0,0 +1,317 @@
+
+<div class="simulator-desktop-container" data-view="/Content/Applications/20_1/GettingStartedWith/SelectBox/index.html, /Content/Applications/20_1/GettingStartedWith/SelectBox/index.js, /Content/Applications/20_1/GettingStartedWith/SelectBox/index.css"></div>
+
+Refer to the subtopics for details on every configuration step. You can also see the full code below or download it from <a href="https://github.com/DevExpress-Examples/getting-started-with-selectbox" target="_blank">this GitHub repository</a>.
+
+---
+##### jQuery
+
+    <!-- tab: index.js -->
+    $(function() {
+        $("#selectBox").dxSelectBox({
+            const data = [{
+                ID: 1,
+                Name: 'Banana',
+                Category: 'Fruits',
+            }, {
+                ID: 2,
+                Name: 'Cucumber',
+                Category: 'Vegetables',
+            }, {
+                ID: 3,
+                Name: 'Apple',
+                Category: 'Fruits',
+            }, {
+                ID: 4,
+                Name: 'Tomato',
+                Category: 'Vegetables',
+            }, {
+                ID: 5,
+                Name: 'Apricot',
+                Category: 'Fruits',
+            }]
+
+            var dataSource = new DevExpress.data.DataSource({
+                store: data,
+                type: 'array',
+                key: "ID",
+                group: "Category"
+            });
+
+            $("#selectBox").dxSelectBox({
+                dataSource: dataSource,
+                valueExpr: "ID",
+                displayExpr: "Name",
+                searchEnabled: "true",
+                onValueChanged: function(e) {
+                    const item = data.filter(i => i.ID === e.value)[0];
+                    console.log(item.ID + ": " + item.Name);
+                },
+                grouped: true
+            });
+        });
+    });
+
+    <!-- tab: index.html -->
+    <html>
+        <head>
+            <!-- ... -->
+            <script type="text/javascript" src="https://code.jquery.com/jquery-3.5.1.min.js"></script>
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/20.1.4/css/dx.common.css">
+            <link rel="stylesheet" href="https://cdn3.devexpress.com/jslib/20.1.4/css/dx.light.css">
+            <script type="text/javascript" src="https://cdn3.devexpress.com/jslib/20.1.4/js/dx.all.js"></script>
+            <script type="text/javascript" src="index.js"></script>
+        </head>
+        <body>
+            <div id="selectBox"></div>
+        </body>
+    </html>
+
+
+##### Angular
+
+    <!-- tab: app.component.html -->
+    <dx-select-box
+        [dataSource]="dataSource"
+        valueExpr="ID"
+        displayExpr="Name"
+        searchEnabled="true"
```

Comment:
```suggestion
        [searchEnabled]="true"
```

You should evaluate boolean value. Otherwise, it will be assigned a string value. E.g. `searchEnabled="false"` leads to `searchEnabled === true`, because `!!"false" === true`